main.tf

```diff
@@ -108,6 +108,33 @@ resource "aws_lambda_function" "this" {
   depends_on = [null_resource.archive, aws_s3_object.lambda_package, aws_cloudwatch_log_group.lambda]
 }
 
+# This resource contains the extra information required by SAM CLI to provide the testing capabilities
+# to the TF application. The required data is where SAM CLI can find the Lambda function source code
+# and what are the resources that contain the building logic.
+resource "null_resource" "sam_metadata_aws_lambda_function_this" {
+  count = local.create && var.create_package && var.create_function && !var.create_layer ? 1 : 0
+  triggers = {
+    # This is a way to let SAM CLI correlates between the Lambda function resource, and this metadata
+    # resource
+    resource_name = "aws_lambda_function.this[0]",
```

Comment:
Please remove comas after each value on the map.

Reply:
done